### PR TITLE
ENH: Add unadjusted flag to polygon backfilling

### PR DESCRIPTION
**Pull request by tibkiss**

### CODE REVIEW

**Review by kalmant — COMMENTED** [review]
The code looks good but there's a conflict and some tests are failing. I'd approve it if those things were ✅

File: contrib/polygon/backfill/backfill.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ var (
 	BackfillM *sync.Map
 )
 
-func Bars(symbol string, from, to time.Time, batchSize int, writerWP *worker.WorkerPool) (err error) {
+func Bars(symbol string, from, to time.Time, batchSize int, unadjusted bool, writerWP *worker.WorkerPool) (err error) {
```
Comment (kalmant) [inline contrib/polygon/backfill/backfill.go:98]: `contrib/polygon/polygon.go` also uses this function but that file was not updated alongside these changes.

**Review by kalmant — CHANGES_REQUESTED** [review]
`contrib/polygon/polygon.go`

File: contrib/polygon/backfill/backfill.go (hunk 1 above)
Comment (tibkiss) [reply contrib/polygon/backfill/backfill.go:98]: very good catch, thanks!

**Comment by tibkiss** [comment]
closing in favor of https://github.com/alpacahq/marketstore/pull/384

**Pull request closed without merging**